ci/start-core.sh

```diff
@@ -4,7 +4,11 @@ set -e
 
 BITCOIN_VERSION=0.20.1
 
-# This should be cached by Travis
+wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add - || exit 1
+sudo apt-add-repository "deb http://apt.llvm.org/xenial/ llvm-toolchain-xenial-10 main" || exit 1
+sudo apt-get update || exit 1
+sudo apt-get install -y libllvm10 clang-10 libclang-common-10-dev || exit 1
+
```

Comment:
Do we need this to build electrs?

Reply:
Oh ya we need this to build electrs, since it's a fresh container and not the same one used for the wasm32 building which also needs the llvm tools installed. Long term I'd rather publish our own docker containers with regtest bitcoind and electrs, then github actions can just spin those up as needed without rebuilding each time.